cli/src/list_invitations.rs

```diff
@@ -0,0 +1,75 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use clap::Args;
+use std::path::PathBuf;
+
+use libparsec::{
+    authenticated_cmds::latest::invite_list::{self, InviteListItem, InviteListRep},
+    get_default_config_dir, InvitationStatus,
+};
+
+use crate::utils::*;
+
+#[derive(Args)]
+pub struct ListInvitations {
+    /// Parsec config directory
+    #[arg(short, long, default_value_os_t = get_default_config_dir())]
+    config_dir: PathBuf,
+    /// Device slughash
+    #[arg(short, long)]
+    device: Option<String>,
+}
+
+pub async fn list_invitations(list_invitations: ListInvitations) -> anyhow::Result<()> {
+    load_device_and_run(
+        list_invitations.config_dir,
+        list_invitations.device,
+        |cmds, _| async move {
+            let handle = start_spinner("Listing invitations");
+
+            let rep = cmds.send(invite_list::Req).await?;
+
+            let invitations = match rep {
+                InviteListRep::Ok { invitations } => invitations,
+                rep => {
+                    return Err(anyhow::anyhow!(
+                        "Server error while listing invitations: {rep:?}"
+                    ));
+                }
+            };
+
+            handle.done();
+
+            if invitations.is_empty() {
+                println!("No invitation.");
+            }
```

Comment:
```suggestion
                println!("No invitation.");
                return Ok(());
            }
```

I would prefer to have an early return or have an `if .. else ..` block